### DISCUSSION TITLE
[每日一水]想问问大家除了零号空洞还有什么方式可以刷丁尼啊？

### DISCUSSION THREAD

**Discussion by Merce-Odyssey** [share121/inter-knot#3015] [其他]
零号空洞每周也就只能刷225k，角色50突破都要400k呢
![无标题](https://github.com/user-attachments/assets/0bbab33c-9d36-4eac-8db7-af1daeb5610a)

**Comment by LT-UT** [comment]
打拿命验收
一次最多80丁尼
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by Merce-Odyssey** [reply]
真是高高又效效啊 (@_@)

**Reply by LT-UT** [reply]
你也可以用体力刷

**Comment by LinkboyTvT** [comment]
把你的信号残响、零号业绩、调查业绩啥的都换丁尼，够用，我开服到现在拉面吃不停也没刷丁尼，前提是你时间有多，这些东西就没那么值钱了
